composer.json

```diff
@@ -18,18 +18,19 @@
     "require": {
         "php": ">=7.4.0",
         "doctrine/cache": "^1.0",
-        "doctrine/common": "^2.6 || ^3.0",
-        "doctrine/dbal": "^2.5 || ^3.0",
-        "doctrine/migrations": "^3.0",
-        "doctrine/orm": "^2.5",
-        "doctrine/persistence": "^1.3 || ^2.0",
+        "doctrine/common": "^3.2",
+        "doctrine/dbal": "^3.3",
+        "doctrine/migrations": "^3.4",
+        "doctrine/orm": "^2.11",
+        "doctrine/persistence": "^2.3",
         "psr/container": "^1.0 || ^2.0"
     },
     "require-dev": {
         "doctrine/annotations": "^1.13",
         "doctrine/coding-standard": "^8.1",
         "phpunit/phpunit": "^9.4",
         "psalm/plugin-phpunit": "^0.9",
+        "psr/cache": "^1.0",
```

Comment:
`psr/cache` is currently at `3.0` - we'd be moving to an older release here :thinking:

Reply:
Yah, also not sure if we actually need this dependency as it is a requirement by doctrine anyways. Will drop it here.